site/components/map/Map.tsx

```diff
@@ -261,14 +361,137 @@ const Map = ({
+                searchedRoute?.outbound ? (
+                    <Source
+                        key={searchedRoute?.serviceId}
+                        id={`outbound-route-${searchedRoute?.serviceId}`}
+                        type="geojson"
+                        data={createLineString(searchedRoute.outbound, searchedRoute?.serviceId)}
+                    >
+                        <Layer
+                            id={`services-outbound-${searchedRoute?.serviceId}`}
+                            type="line"
+                            source={`services-outbound-${searchedRoute?.serviceId}`}
+                            layout={lineLayout}
+                            paint={lineStyle}
+                        />
+                        <Layer
+                            id={`services-highlighted-outbound-${searchedRoute?.serviceId}`}
+                            type="line"
+                            source={`services-outbound-${searchedRoute?.serviceId}`}
+                            layout={lineLayout}
+                            paint={lineStyleHighlight}
+                            filter={filter}
+                        />
+                    </Source>
+                ) : null,
+            ),
+        [filter],
+    );
+
+    const getInteractiveLayerIds = useCallback(
+        () =>
+            searchedRoutes && searchedRoutes.length > 0
+                ? searchedRoutes.flatMap((sr) => {
+                      if (sr?.inbound && sr.outbound) {
+                          return [
+                              `services-inbound-${sr?.serviceId || ""}`,
+                              `services-outbound-${sr?.serviceId || ""}`,
+                          ];
+                      } else if (sr?.inbound) {
+                          return [`services-inbound-${sr?.serviceId || ""}`];
+                      } else if (sr?.outbound) {
+                          return [`services-outbound-${sr?.serviceId || ""}`];
+                      } else {
+                          return [];
+                      }
+                  })
+                : [],
+        [searchedRoutes],
+    );
+
+    const getServiceInfo = (id: number) => {
+        const service = services ? services.find((service) => service.id === id) : null;
+        return service
+            ? `Line: ${service.origin.replace("_", " ")} - ${service.destination.replace("_", " ")}`
+            : "Line: N/A";
+    };
     return mapboxAccessToken ? (
         <>
+            {selected.length === 100 ? (
+                <div className="govuk-warning-text">
+                    <span className="govuk-warning-text__icon" aria-hidden="true">
+                        !
+                    </span>
+                    <strong className="govuk-warning-text__text">
+                        <span className="govuk-warning-text__assistive">Warning</span>
+                        {`Stop selection capped at 100, ${selected.length || "0"} stops currently selected`}
```

Comment:
I think ${selected.length || "0"} can just be selected.length as its an array thats passed in, it will always have a length.

Reply:
done